frontend/src/components/pages/teacher/DistributeAssessmentPage.tsx

```diff
@@ -1,8 +1,247 @@
-import React from "react";
-import { Text } from "@chakra-ui/react";
+import React, { useMemo } from "react";
+import { useHistory } from "react-router-dom";
+import {
+  Box,
+  Button,
+  Center,
+  HStack,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Text,
+} from "@chakra-ui/react";
+
+import { PlusOutlineIcon } from "../../../assets/icons";
+import DistributeAssessmentsIllustration from "../../../assets/illustrations/distribute-assessments.svg";
+import { titleCase } from "../../../utils/GeneralUtils";
+import TestSessionListItem, {
+  STATUSES,
+  TestSessionStatus,
+} from "../../assessments/TestSessionListItem";
+import ErrorState from "../../common/ErrorState";
+import LoadingState from "../../common/LoadingState";
+import MessageContainer from "../../common/MessageContainer";
+import Pagination from "../../common/table/Pagination";
+
+const CREATE_ASSESSMENT_PATH = "/teacher/assessments/create";
+
+const mockData: {
+  id: string;
+  accessCode: string;
+  classroomName: string;
+  stats?: {
+    mean: number;
+    median: number;
+    completionRate: number;
+    submissions: number;
+  };
+  startDate: Date;
+  endDate: Date;
+  testName: string;
+}[] = [
+  ...[...Array(20)].map((_, i) => ({
+    id: i.toString(),
+    accessCode: "086 731",
```

Comment:
if you fetch the access codes, i think they probably don't have a space in the middle & design added a subsequent space for readability. so you would have to add the space on the fe